client/cmd/transaction.go

```diff
@@ -76,13 +76,13 @@ func buildTransactionCommand() {
 		Short: "eris-client tx call --amt <amt> --fee <fee> --gas <gas> --to <contract addr> --data <data>",
 		Long:  "eris-client tx call --amt <amt> --fee <fee> --gas <gas> --to <contract addr> --data <data>",
 		Run:   func(cmd *cobra.Command, args []string) {
-			// transaction.Call(clientDo)
+			transaction.Call(clientDo)
 		},
 		PreRun: assertParameters,
 	}
 	callCmd.Flags().StringVarP(&clientDo.AmtFlag, "amt", "a", "", "specify an amount")
 	callCmd.Flags().StringVarP(&clientDo.ToFlag, "to", "t", "", "specify an address to send to")
-	callCmd.Flags().StringVarP(&clientDo.DataFlag, "data", "d", "", "specify some data")
+	callCmd.Flags().StringVarP(&clientDo.DataFlag, "data", "", "", "specify some data")
```

Comment:
If you're killing the shorthand name, shouldn't you be using `StringVar`. Looking at the source I see this has the same effect (https://github.com/ogier/pflag/blob/master/string.go#L34) but you intention would be clearer and you would not be exposed to a change of implementation giving you undefined behaviour if you used `StringVar`